hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -118,6 +118,34 @@ inline void CLR_GPRS_TIMEOUT() {
     DEBUG("GPRS WD Cleared, was %d", gprs_timeout_duration);
 }
 
+namespace {
+
+AcT toCellularAccessTechnology(int v) {
+    switch (v) {
+        case 0: // GSM
```

Comment:
Instead of the magic numbers, can we pull [`CellularAccessTechnology`](https://github.com/particle-iot/device-os/blob/develop/hal/src/boron/network/cellular_ncp_client.h#L51) into a shared header and use the enumeration instead?

Reply:
Honestly I think the magic numbers are fine, but if we wanted to supply an enum it should be one for the modems used like `UbloxCellularAccessTechnology`.  Down the road we will have to likely map the modem's codes to `CellularAccessTechnology` and also to this used in enums_hal.h
```
//! Access Technology
typedef enum {
    ACT_UNKNOWN     = 0,
    ACT_GSM         = 1,
    ACT_EDGE        = 2,
    ACT_UTRAN       = 3,
    ACT_CDMA        = 4,
    ACT_LTE         = 5,
    ACT_LTE_CAT_M1  = 6,
    ACT_LTE_CAT_NB1 = 7
} AcT;
```

Reply:
And then the enum makes it a bit harder to use `toCellularAccessTechnology()`

Reply:
I agree about the magic numbers, might as well fix that. Resolved in https://github.com/particle-iot/device-os/commit/5c168b39a4b5d90a94062d5fdc1410457e909e92